sycl/test/extensions/properties/properties_kernel.cpp

```diff
@@ -18,6 +40,78 @@ int main() {
   static_assert(
       is_property_value<decltype(work_group_size_hint<6, 6, 6>)>::value);
   static_assert(is_property_value<decltype(sub_group_size<7>)>::value);
+  static_assert(is_property_value<decltype(sub_group_size<7>)>::value);
```

Comment:
That's a lot of similar lines. Can we introduce some helper function in here (perhaps with variadic templates help?) to simply iterate over all those aspects (they could be hidden into a macro)?

Reply:
I have done some refactoring to the test. Is this what you had in mind?

Reply:
The new version looks better, thanks!